firmware/port.c

```diff
@@ -140,7 +152,7 @@ void port_disable(PortData* p) {
 
 void port_send_status(PortData* p, u8 d) {
     if (p->reply_len >= BRIDGE_BUF_SIZE) {
-        bridge_disable_chan(p->chan);
+        port_error(p);
```

Comment:
And these functions might eventually want to go in a `port-platform.c`, and also abstract out IO operations for non-SAMD21 hardware like Reach.